packages/tests/package.json

```diff
@@ -12,9 +12,10 @@
     "@types/prettier": "^2.0.1",
     "@types/diff": "^5.0.0",
     "@types/fs-extra": "^9.0.1",
+    "@web/dev-server": "=0.1.30",
     "@web/dev-server-legacy": "^0.1.7",
-    "@web/dev-server-rollup": "^0.3.13",
-    "@web/test-runner": "^0.13.22",
+    "@web/dev-server-rollup": "=0.3.15",
```

Comment:
I wasn't able to find docs around this syntax. Is this a syntax to pin that exact version? Or a version of "save-exact"?

Reply:
It means exactly equal: https://github.com/npm/node-semver#ranges